mock/api.js

```diff
@@ -6,41 +6,24 @@ const DEFAULT_SIZE = 100;
 export const mockIndices = new Array(DEFAULT_SIZE).fill().map(() => moment().format('YYYY-MM'));
 
 export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-  controller: casual.word + index,
-  key: casual.word + index,
-  last_modified_string: moment.utc() + index,
+  controller: `${casual.word}.${casual.word}.com`,
+  key: casual.ip,
+  last_modified_string: moment.utc(),
   last_modified_value: moment.utc() + index,
```

Comment:
These are supplied by Elasticsearch as part of the aggregation. The query aggregates results by the `"run.start"` field. It's a date datatype, and Elasticsearch apparently stores it internally as a number; presumably a UNIX seconds (or milliseconds) since the Epoch, though I never bothered to work that out. It returns both that binary internal representation and the string date. Both were passed up from the model before, and I left it that way ... although I'm not at all sure the dashboard code actually cares.

This sort of thing would be good to consider when we think about the additions you want to make; we don't have to be tied to the "raw" Elasticsearch data where that doesn't help us.

Reply:
According to [EpochConverter](https://www.epochconverter.com/) (which is my go-to tool for this stuff), that's milliseconds since the UNIX Epoch, which is Midnight, 1 January 1970 UTC.

It probably should be an integer (although, fractional milliseconds _do_ exist ;-) ), but apparently it somehow got converted to a float somewhere in the many layers between there and here.

Reply:
I see Elasticsearch returning the timestamp in normalized exponential form: `"value": 1.617290619102E12`. The Pbench `/controllers/list` API just copies the value.

Reply:
I remember chasing all of this stuff down in a previous life.  ES has its own foibles, and then that's compounded by the requirements of JSON during transport, and then it's all subject to the fact that the JSON gets converted to a Python 'dict' with Python datatypes...so, who knows?!

Reply:
Fixes it to use milliseconds since the UNIX Epoch